frontend/packages/dev-console/src/components/monitoring/MonitoringDashboard.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+
+const MonitoringDashboard: React.FC<{}> = () => {
```

Comment:
nit : can we just have `React.FC` instead of `React.FC<{}>`

Reply:
Fixed